RpcWallet/RpcWallet.cs

```diff
@@ -313,29 +313,36 @@ private JObject SendFrom(UIntBase assetId, UInt160 from, UInt160 to, string valu
 
             ContractParametersContext transContext = new ContractParametersContext(tx);
             Wallet.Sign(transContext);
-            tx.Witnesses = transContext.GetWitnesses();
-            if (tx.Size > 1024)
+            if (transContext.Completed)
             {
-                Fixed8 calFee = Fixed8.FromDecimal(tx.Size * 0.00001m + 0.001m);
-                if (fee < calFee)
+                tx.Witnesses = transContext.GetWitnesses();
+                if (tx.Size > 1024)
```

Comment:
Erik, when I see this hard codded parameter I come back with that discussion about the Simple Policy as a native contract.